pkgs/by-name/ls/lsh/package.nix

```diff
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "lsh";
+  version = "1.3.3";
+  src = fetchFromGitHub {
+    owner = "latitudesh";
+    repo = "lsh";
+    rev = "v${version}";
+    sha256 = "0YpjG4u+wb4LRWzfTddKFwut0MBzEch+HZijmZiVXpE=";
+  };
+  vendorHash = "sha256-ogdyzfayleka4Y8x74ZtttD7MaeCl1qP/rQi9x0tMto=";
+  subPackages = [ "." ];
+  meta = with lib; {
+    description = "Command-Line Interface for Latitude.sh";
+    homepage = "https://github.com/latitudesh/lsh";
+    license = licenses.mit;
+    changelog = "https://github.com/latitudesh/lsh/releases/tag/v${version}";
```

Comment:
You need to add yourself as maintainer for new packages.

I am not sure about the package name suddenly pointing to a new (completely different) package. Probably okay, since `lsh` was removed for one NixOS release.

Reply:
Gap when there is no tool under such name in stable at least half year.